src/libraries/Directory.Build.targets

```diff
@@ -116,8 +116,8 @@
 
   <Target Name="ReportConfigurationErrorMessage"
           BeforeTargets="AssignProjectConfiguration"
-          Condition="'$(BuildConfigurations)' != ''" >
-    <Message Importance="Low" Text="$(MSBuildProjectFullPath), C: $(Configuration) BC: $(BuildConfiguration) BCs: $(BuildConfigurations)" />
+          Condition="'$(TargetFrameworks)' != ''" >
+    <Message Importance="Low" Text="$(MSBuildProjectFullPath), T: $(TargetFrameowork) BS: $(BuildSettings) Tfms: $(TargetFrameworks)" />
     <Message Importance="High" Condition="'$(ConfigurationErrorMsg)' != ''" Text="$(MSBuildProjectFullPath) ConfigurationErrorMessage: $(ConfigurationErrorMsg)" />
```

Comment:
Is `ConfigurationErrorMsg` still defined in the new configuration system? Looks like this is dead code, I would remove the whole target. Also typo above: `$(TargetFrameowork)`